src/callbacks.c

```diff
@@ -0,0 +1,204 @@
+
+#include "callbacks.h"
+#include "main.h"
+#include "cx.h"
+#include "os_io_seproxyhal.h"
+#include "os.h"
+#include "ssh_ux.h"
+
+unsigned int io_seproxyhal_touch_exit(const bagl_element_t *e) {
+   (void) e;
```

Comment:
`e` is never used in any of this callback and could therefore be remove in another commit maybe?